client/allocrunner/networking_bridge_linux.go

```diff
@@ -195,10 +225,16 @@ const nomadCNIConfigTemplate = `{
 						{
 							"subnet": %q
 						}
+					],
+					[
+						{ 
+							"subnet": %q
+						}
 					]
 				],
 				"routes": [
-					{ "dst": "0.0.0.0/0" }
+					{ "dst": "0.0.0.0/0" },
+                    { "dst" : "::/0" }
```

Comment:
nitpick: indentation inconsistency between these
```suggestion
					{ "dst": "0.0.0.0/0" },
					{ "dst" : "::/0" }
```